src/Command/Ingester/LineItemIngesterCommand.php

```diff
@@ -172,6 +173,7 @@ private function createLineItem(array $rawLineItem): LineItem
     {
         $lineItem =
             (new LineItem())
+                ->setId(new UuidV6())
```

Comment:
Any idea of performance impact?

Reply:
Do we have to check The increase of the database (Index, data)? Does it imply more costs?

Reply:
I have a more interesting question, not for this file, but for the whole PR itself:
what is the reason to go with UUIDv6, instead of ULID (though both are supported by 5.x branch of symfony as I see)?
Because UUIDv6 is not part of the standard, and status of RFC (?) is undefined (at least I didn't manage to find recent info on status).
So for me it's more a question of: should we really go with smth non-standard (though supported) or with smth "standard"?
p.s.
As I can see with my google-fu:
- https://symfony.com/doc/current/components/uid.html#ulids
- https://github.com/prooph/event-store/issues/361 (just the discussion around using ULIDs)
p.p.s
Though I didn't find it - I though that there already was a some kind of discussion either in slack or elsewhere on ULIDs limitations and what to use, maybe you can point me

Reply:
I think we discussed this today on Slack. Can I close this conversation @uncleempty ?

Reply:
Resolving the conversation, if you think we need more discussion around this, let's reopen it.